databricks/koalas/tests/test_frame_plot.py

```diff
@@ -201,3 +214,12 @@ def test_missing(self):
             with self.assertRaisesRegex(PandasNotImplementedError,
                                         "method.*DataFrame.*{}.*not implemented".format(name)):
                 getattr(ks.plot, name)()
+
+    def test_topn_max_rows(self):
+        self.setUpClass()
```

Comment:
Seems we can remove this since `setUpClass` will be called when this test class is started.

Reply:
maybe this should stay? since only for this test, `max_rows` will be set to 2000? @HyukjinKwon

Reply:
i see, will remove